.pytype.cfg

```diff
@@ -0,0 +1,23 @@
+[pytype]
+exclude =
+    **/*_test.py
+    **/test_*.py
+# Space-separated list of files or directories to process.
+inputs =
+    .
+# Keep going past errors to analyze as many files as possible.
+keep_going = False
```

Comment:
Should we treat this like a test runner, where every failure must be fixed, or like a syntax checker, where some violations might valid and should be ignored?

Reply:
since this is on stickler I assume this will allow it to analyse all files in a PR instead of stopping at the first issue.

Reply:
`keep_going = False` seems like it would stop at the first issue. That seems like the opposite of what you said will happen. I'm not following your logic.

Reply:
```suggestion
keep_going = True
```